contacktbook/DataBase.cs

```diff
@@ -29,6 +30,10 @@ public void createNewDatabase()
                                                        tel1 VARCHAR(20), tel2 VARCHAR(20), email VARCHAR(30))";
                     SQLiteCommand createTable = new SQLiteCommand(createContactTableQuery, connection);
                     createTable.ExecuteNonQuery();
+                    // Creating notation to Log file

```

Comment:
It is worth to add documentation comment in LoggerClass.WriteLog() method. In this case we'll see comments on Mouse flow in every place we invoke it.